Base/Logger/Logger.Static/FileLogger.hpp

```diff
@@ -70,30 +70,39 @@ class FileLogger : ILogger
      */
     FileLogger& operator=(const FileLogger&) = delete;
 
-    /*
-    * @brief Method used to say that's going to be an error
-    * @param const std::string& -> incoming message, that should describe situation or progress
-    */
+    /**
+     * @brief Method used to say that's going to be an error
+     * @param const std::string& -> incoming message, that should describe situation or progress
+     */
     void error(const std::string& message);
 
-    /*
-    * @brief Method used to say that's going to be a possible error
-    * @param const std::string& -> incoming message, that should describe situation or progress
-    */
+    /**
+     * @brief Method used to say that's going to be a possible error
+     * @param const std::string& -> incoming message, that should describe situation or progress
+     */
     void warning(const std::string& message);
 
-    /*
-    * @brief Method used to say general information
-    * @param const std::string& -> incoming message, that should describe situation or progress
-    */
+    /**
+     * @brief Method used to say general information
+     * @param const std::string& -> incoming message, that should describe situation or progress
+     */
     void info(const std::string& message);
 
-    /*
-    * @brief Method used to say general information detailing debug progress
-    * @param const std::string& -> incoming message, that should describe situation or progress
-    */
+    /**
+     * @brief Method used to say general information detailing debug progress
+     * @param const std::string& -> incoming message, that should describe situation or progress
+     */
     void debug(const std::string& message);
 
+    /**
+     * @brief Method used to say some information and to throw an exception
+     * @param const char* -> name of file where the event took place
+     * @param std::uint16_t -> line of the file where event took place
```

Comment:
About file and line, look at __LINE__ and __FILE__ macros.
Best option will be to make some alias with predefined values as a parameters.

Reply:
I tried to do it. Check please.
I think we need discuss this topic at the daily meeting.